examples/tv-casting-app/darwin/TvCasting/TvCasting/CommissioningViewModel.swift

```diff
@@ -19,6 +19,39 @@
 import Foundation
 import os.log
 
+class ExampleDAC : DeviceAttestationCredentialsHolder {
```

Comment:
it might be nice for this class to be separated out into a separate file similar to the Android one

Reply:
Done.